distrobox-init

```diff
@@ -458,6 +458,16 @@ if ! grep -q "${container_user_name}" /etc/passwd; then
 		--gid "${container_user_gid}" \
 		"${container_user_name}"; then
 
+		printf "Warning: there was a problem setting up the user\n"
+	fi
+else
```

Comment:
Let's just add a couple of comments here
this situation is presented when podman or docker already creates the user for us, so we find it already present